rotkehlchen/chain/evm/decoding/hop/constants.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+from typing import Final
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.types import ChecksumEvmAddress
+
+
+@dataclass
+class HopBridge:
+    address: ChecksumEvmAddress  # https://github.com/hop-protocol/hop/blob/d842c19bacff7dbe0b2fda71763f5462a943a146/packages/sdk/src/addresses/mainnet.ts
+    asset: Asset
+    amm_wrapper: ChecksumEvmAddress | None = None
```

Comment:
this is not a constant. Add a docstring explaining explaining what each field is and move it to a types files or similar